site/content/kapp-controller/docs/develop/packaging.md

```diff
@@ -309,3 +309,39 @@ status:
 
 **Note:** Values will only be included in the first templating step of the package,
 though we intend to improve this experience in later releases.
+
+## Combinations and Special Cases
+
+### Overlapping Package Repositories
```

Comment:
lets add "Available as of x" (e.g. https://github.com/vmware-tanzu/carvel/blob/develop/site/content/kbld/docs/develop/config.md?plain=1#L324)